.env.bak

```
@@ -0,0 +1,115 @@
+############
+# N8N Configuration
+############
+N8N_ENCRYPTION_KEY=$(openssl rand -hex 24)
+N8N_USER_MANAGEMENT_JWT_SECRET=$(openssl rand -hex 32)
+
+############
+# Supabase Secrets
+############
+POSTGRES_PASSWORD=$(openssl rand -base64 32 | tr -d '/+=' | head -c 32)
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)